### PR TITLE
ews mail sender: support old exchange servers

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33096

## Description
Fix support for older ews servers that still use tls1.0.
 
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EWSMailSender/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### EWS Mail Sender
+- Fixed a regression were the integration was failing to connect to old versions of Exchange, which are still using TLS1.0.
```
Comment (anara123) [inline Packs/EWSMailSender/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Fixed a regression where the integration was failing to connect to old versions of Exchange, which are still using TLS1.0.
```

**Review by anara123 — COMMENTED** [review]
where is the fix ? in the dockerimage update?

**Comment by glicht** [comment]
The docker is the fix. See https://github.com/demisto/dockerfiles/pull/3843

**Review by anara123 — APPROVED** [review]

File: Packs/EWSMailSender/pack_metadata.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  "email": "",
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Messaging"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
+    "name": "EWS Mail Sender",
+    "description": "Exchange Web Services and Office 365 Email Sender. Note: this Integration supports Office 365 basic authentication only. If you are using Office 365, we recommend using the EWS O365 Integration instead, which supports modern authentication (oauth2).",
```
Comment (glicht) [inline Packs/EWSMailSender/pack_metadata.json:3]: @amshamah419 can you review this text. Thanks.

File: Packs/EWSMailSender/ReleaseNotes/1_1_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### EWS Mail Sender
+- Fixed a regression where the integration was failing to connect to old versions of Exchange, which are still using TLS1.0.
```
Comment (amshamah419) [inline Packs/EWSMailSender/ReleaseNotes/1_1_1.md:4]:
```suggestion
- Fixed a regression where the integration was failing to connect to older versions of Exchange, which still use TLS1.0.
```

**Review by amshamah419 — COMMENTED** [review]
Description looks good. See changes for RNs.

**Pull request merged**